runners/flink/job-server-container/Dockerfile

```diff
@@ -19,6 +19,8 @@
 FROM openjdk:8
 MAINTAINER "Apache Beam <dev@beam.apache.org>"
 
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y libltdl7
```

Comment:
Is this required for `which`?

Reply:
no, this in required for docker. I suppose this is linux or atleast google linux related things where host mounted docker needs libltdl7.